tests/client/amo/testApi.js

```diff
@@ -216,9 +211,9 @@ describe('amo.api', () => {
         .returns(Promise.resolve({ results: [] }));
 
       return getLatestUserReview()
-        .then(unexpectedSuccess, (error) => {
-          assert.match(error.message, /user and addon must be specified/);
-        });
+        .then(unexpectedSuccess, (error) =>
```

Comment:
I'd like to see braces around a statement like this because it's a common need to add more assertions later. If you remove the return rule you could remove the braces.